vdr/vdr_test.go

```diff
@@ -565,3 +691,67 @@ type roundTripperFunc func(*http.Request) (*http.Response, error)
 func (fn roundTripperFunc) RoundTrip(r *http.Request) (*http.Response, error) {
 	return fn(r)
 }
+
+type testTransaction struct {
```

Comment:
the only test using this struct also passes without specifying a return value from the mock. saves 60+ lines

Reply:
future PRs will need it anyway